rfcs/framework-0036-http-cache-response-headers-records-rest-apis.md

```diff
@@ -0,0 +1,66 @@
+- Start Date: 2020-07-22
+- RFC PR: [#36](https://github.com/inveniosoftware/rfcs/pull/36)
+- Authors: Nicola Tarocco
+- State: DRAFT
+
+# HTTP Cache response headers when fetching records using REST APIs
+
+## Summary
+
+When fetching a record via REST APIs using the endpoint provided by `invenio-records-rest`, Invenio will return the response and a set of HTTP cache headers. In particular, it will return the headers `ETag` and `Last-Modified` that help the client to correctly cache the content.
```

Comment:
Question: the RFC talks about `invenio-records-rest`. Does it apply to other modules with REST endpoints? accounts, files, communities? In the end any page/resource can be cached no? Not sure how feasible is the etag and last-modified attrs in those other resources.

Reply:
Very good point, I am not familiar with `communities` but it probably applies there. Basically it applies to all `GET` request when there is no `Cache-Control` set.

Reply:
Ping @slint @Glignos